sklearn/compose/_column_transformer.py

```diff
@@ -368,41 +374,31 @@ def set_params(self, **kwargs):
         self._set_params("_transformers", **kwargs)
         return self
 
-    def _iter(self, fitted, replace_strings, column_as_strings):
+    def _iter(self, fitted, column_as_labels, skip_drop, skip_empty_columns):
         """
         Generate (name, trans, column, weight) tuples.
 
+
         Parameters
         ----------
         fitted : bool
             If True, use the fitted transformers (``self.transformers_``) to
             iterate through transformers, else use the transformers passed by
             the user (``self.transformers``).
 
-        replace_strings : bool
-            If True, replace 'passthrough' with the fitted version in
-            ``self._name_to_fitted_passthrough``. If True, also skip ``drop``
-            transformers.
-
-        column_as_strings : bool
+        column_as_labels : bool
             If True, columns are returned as strings. If False, columns are
             returned as they were given by the user. This can only be True if
             the ``ColumnTransformer`` is already fitted.
+
+        skip_drop : bool
+            If True, 'drop' transformers are filtered out.
+
+        skip_empty_columns : bool
+            If True, transformers with empty selected columns are filtered out.
```

Comment:
Could we also have `Yields` section to show that this is a generator.

Reply:
I tried adding something, not sure if you like it.

Reply:
Yep this is fine with me.